benchmarks/run_release_benchmark.py

```diff
@@ -0,0 +1,212 @@
+import boto3
+import click
+import os
+import paramiko
+import time
+
+
+def get_boto_client(service_name):
+    client = boto3.client(
+        service_name,
+        region_name='us-east-1',
+    )
+    return client
+
+
+def get_boto_resource(service_name):
+    client = boto3.resource(
+        service_name,
+        region_name='us-east-1',
```

Comment:
Please dont hard code regions. Ensure that `AWS_DEFAULT_REGION` env is set.

Reply:
Done.